python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -990,6 +990,16 @@ class BuiltinDataset(OnDiskDataset):
             Self edges are added to the original graph.
             Node features are stored as float32.
 
+    **igb-het-[tiny|small|medium]**
+        The igb-hom-[tiny|small|medium] dataset is a heterogeneous citation network,
+        which is designed for developers to train and evaluate GNN models with
+        high fidelity. See more details in `igb-het-[tiny|small|medium]
+        <https://github.com/IllinoisGraphBenchmark/IGB-Datasets>`_.
+
+        .. note::
+            Reverse paper__cites__paper edges are added to the original graph.
+            Node features are stored as float32.
```

Comment:
Does adding reverse edges help the accuracy? If not, let's add it only for the MLPerf version of the dataset, named `igb-het-mlperf`. Let's also use float16 dtype for the features of `igb-het-mlperf`. Note that there will still be `igb-het`.